plugins/docker/s.ml

```diff
@@ -83,6 +83,11 @@ module type DOCKER = sig
 
   val service : name:string -> image:Image.t Current.t -> unit -> unit Current.t
   (** [service ~name ~image ()] keeps a Docker SwarmKit service up-to-date. *)
+
+  val compose : name:string -> image:Image.t Current.t -> contents:string -> unit -> unit Current.t
+  (** [service ~name ~image ~contents ()] keeps a Docker Compose deployment up-to-date.
+      [contents] contains the full Compose Yaml file, and the [$IMAGE_HASH] variable will be
+       substituted at deployment time with the SHA256 hash to pull. *)
```

Comment:
Probably we should just have `contents:string Current.t` here and let the caller to whatever regex stuff they want. Otherwise, this is limited to composing a single service.

Reply:
Done